src/lib/y2network/presenters/interfaces_summary.rb

```diff
@@ -50,6 +53,115 @@ def text
 
         Yast::Summary.DevicesList(overview)
       end
+
+      # Generates a one line text summary for the configured interfaces.
+      #
+      # @example with one configured interface
+      #   presenter.one_line_text
+      #   => "DHCP / eth1"
+      #
+      # @example with many configured interfaces
+      #   presenter.one_line_text
+      #   => "Multiple Interfaces"
+      #
+      # @return [String] summary in just one line and in plain text
+      def one_line_text
+        protocols = []
+        output = []
+
+        protocols << "DHCP" if config.connections.any?(&:dhcp?)
+        protocols << "STATIC" if config.connections.any?(&:static?)
+
+        output << protocols.first if protocols.uniq.size == 1
+
+        case config.connections.size
+        when 0
+          return Yast::Summary.NotConfigured
+        when 1
+          output << config.connections.first.interface
```

Comment:
Why not asking here for the protocol?

```ruby
connection = config.connections.first
protocol = "DHCP" if connection.dhcp?
protocol ||= "STATIC" if connection.static?

[protocol, connection.interface].compact.join(" / ")
```

Reply:
Probably can be written in a better way but the procol is also used in case there are multiple connections using the same one. That is, it can be like:
- `DHCP / eth0`: Only one with DHCP
- `STATIC / eth1`: Only one configured statically
- `DHCP / Multiple Interfaces`: More than one but all sharing the dhcp protocol
- `STATIC / Multiple Interfaces`: More tan one but all configured statically
- `Multiple Interfaces`: More than one with different protocols